src/browser/AppInit.jsx

```diff
@@ -82,6 +86,26 @@ bus.applyMiddleware((_, origin) => (channel, message, source) => {
   store.dispatch({ ...message, type: channel, ...origin })
 })
 
+async function setupSentry() {
+  let isCanary = false
```

Comment:
The canary check can be skipped if we're not in production (i.e. move it inside the `if` block below)

Reply:
True, I'll do so!